RummikubGameApp/RummikubGameApp/Tile.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace RummikubGameApp
+{
+    public class Tile
+    {
+        public string Color { get; set; }
+        public int Value { get; set; }
+

```

Comment:
doesn't need set since the value will probably stay the same, will change to work with all args ctor